package.json

```diff
@@ -57,6 +57,11 @@
     "rebuild": "npm run rebuild-changes",
     "build": "npm run build-changes",
     "build:arm": "npm run build-changes:arm",
+    "build:android": "npm ci --ignore-scripts; node scripts/build-android.js",
+    "build:ios": "npm ci --ignore-scripts; scripts/build-ios.sh",
+    "build:node-x64": "npm run package -- --arch=x64",
+    "build:node-ia32": "npm run package -- --arch=ia32",
+    "build:node": "npm run package",
```

Comment:
Why not call `cmake-js` or `prebuilt` directly? It feels that we have multiple script doing almost the same thing.

Reply:
Can you please check the latest commits and see if they are clearer now?